### ISSUE TITLE
ShiftControlM no longer an option? Can't send to Continue

### ISSUE THREAD

**Issue by ewebgh33** [continuedev/continue#989]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS:Windows11
- Continue:latest (just updated) 0.8.2
- IDE:VSCode
```

### Description

Opened VSCode for the first time in about a week.
Continue was no longer in the side panel where I had docked it.
Opened extensions... saw it wanted an update. Updated.
Restart VSCode.
Still no continue. Can't Control+Shit+M to send code to explain any more.
When I highlight code, there is no longer a popup tip telling me I can send to continue.
Is this supposed to be doing this?
Docs/release notes don't say anything about this behaviour.

Basically I can copy/paste into Continue but all the keyboard shortcut functionality is gone and the main github page doesn't have a link to release notes or anything so simple to find as that, so I don't know if this is supposed to be changed or not.

### To reproduce

To reproduce, uh, try to use it?
Control+Shift+M no longer does anything. I can't tell it to /edit.
I CAN see the Continue window again now, but I can't send any blocks of highlighted code to it.

**Comment by sestinj** [comment]
@ewebgh33 we've switched to the shortcuts cmd/ctrl+shift+L and cmd/ctrl+L instead of M. Continue is intended to bring up the tutorial again but if it didn't you can always view this and keyboard shortcuts through the help page in the sidebar, or VS Code's keyboard shortcuts page

**Comment by ewebgh33** [comment]
Thanks that explains it
Definitely didn't bring up the tut again
And didn't see a mention of this when updated in VSCode. No wonder I was confused.

**Comment by sestinj** [comment]
Ok, very good to know! This was exactly the confusion we were *hoping* to avoid : / I'm going to close the issue but will double check the tutorial so others don't experience the same

**Comment by ChildOf7Sins** [comment]
![image](https://github.com/continuedev/continue/assets/131901009/badb41d0-6f99-42d6-b0f2-ea38995b5fc1)
Some prompt somewhere is still saying M fyi

**Comment by finder39** [comment]
@sestinj the website also still states to use the old shortcut. This issue was the only way I was able to figure out to do something different

https://continue.dev/docs/how-to-use-continue
